### ISSUE TITLE
Create document of Terms, Concepts, and Definitions

### ISSUE THREAD

**Issue by robnist** [usnistgov/metaschema-java#246]
Create a document that defines the terms, concepts, definitions, relationships, etc. that are to be used in the project. Other requirements and documents must use these terms and concepts.

Terms and Concepts to include:

Conformance
Compliance
Compatibility
Interoperability
Use Case
Actor
Capability
Data Interface Requirement
Functional Requirement
Obligation
Obligation Code Set
Profile
Implementation Guide
Interaction
Transaction
and so on.

The list of terms and concepts will be expanded.

**Comment by david-waltermire** [comment]
@robnist  Most of these terms are not used in this project.

In this repo and the [related website](https://pages.nist.gov/metaschema), we have tried to avoid creating any new terms. We are focusing on the use of plain language as much as possible. The philosophy used here is to avoid terms, the use of which often leads to confusion and incorrect assumptions. Instead we have focused on using common dictionary words.

The exception to this are some [Metaschema terms](https://pages.nist.gov/metaschema/specification/glossary/), which are used sparingly when needed in this project.

I'd like to focus your feedback on specific areas of text that need clarification. Is there text in this repository or website that you find confusing? If so, how and what should we explain to clarify?
